code/modules/events/wizard/aid.dm

```diff
@@ -36,7 +36,8 @@
 				S.spell_level++
 				if(S.spell_level >= 6 || S.charge_max <= 0) //Badmin checks, these should never be a problem in normal play
 					continue
-				S.charge_max = round(initial(S.charge_max) - S.spell_level * (initial(S.charge_max) - S.cooldown_min)/ S.level_max)
+				if(S.level_max > 0)
+					S.charge_max = round(initial(S.charge_max) - S.spell_level * (initial(S.charge_max) - S.cooldown_min) / S.level_max)
```

Comment:
Instead of proc trying to do things with spell that have level_max less than 1 just make it skip this spell completely and proceed with next spell